build/broad_sanger/03a-nci60Drugs.py

```diff
@@ -39,7 +52,7 @@ def main():
     if not os.path.exists('DOSERESP.csv'):
```

Comment:
I assume this file was already unzipped and in the filepath, but if not, the name of the zipped file should be DOSERESP.zip.